package.json

```diff
@@ -4,7 +4,13 @@
   "description": "Transforms arrays into virtual dom trees",
   "main": "index.js",
   "module": "index.js",
+  "scripts": {
+    "test": "tead"
```

Comment:
Why not adding code coverage as well :`tead --coverage` ?

Reply:
Will do!

Reply:
Wait.. that wasn't as simple as it sounded 😅 
```
$ tead --coverage
/bin/sh: npx: command not found
Done in 0.22s.
```

Reply:
Are you using Node.js < 8?

See the note here: https://github.com/teadjs/tead#npx

Reply:
I'm on `v8.1.3` but I am not using npx anywhere in the project, or at all in fact

Reply:
Sorry, forgot that`npx` wasn't introduced in Node.js 8.0.0, but comes bundled with [`npm` >= 5.2.0](https://github.com/npm/npm/releases/tag/v5.2.0), which comes bundled with [Node.js >= 8.2.0](https://github.com/nodejs/node/pull/14235). 😂 

It's not like I've [already posted about this](https://github.com/hyperapp/hyperapp/issues/518#issuecomment-355816185) and forgot or anything 🙄 

`npx` is a pretty cool tool that allows you to run a command from any package on npm without having to add it as a dependency, but the user still has control over which version they run.   First it checks if the package is installed in `node_modules`, then globally, and finally it just downloads a copy of the latest version on-demand from npm as a last resort. 😎

Reply:
FYI:
![screen shot 2018-01-27 at 13 32 04](https://user-images.githubusercontent.com/3735164/35476541-92d8abf4-0366-11e8-8e5d-0e663025b970.png)
![screen shot 2018-01-27 at 13 32 17](https://user-images.githubusercontent.com/3735164/35476542-98f61f4e-0366-11e8-9864-df8d0fea5ac0.png)